app/views/support_interface/providers/index.html.erb

```diff
@@ -10,6 +10,12 @@
       </tr>
     </thead>
 
+    <% if @filter.search_results_filtered_out_count.positive? %>
+      <p class="govuk-body">
+      <%= pluralize(@filter.search_results_filtered_out_count, 'search result') %> were filtered out.
```

Comment:
This pluralise doesn't read well if it's singular, maybe drop the "were"?

Reply:
oops!